File: RepoTransBench/target_projects/Java/C#/JakeWharton_picasso2-okhttp3-downloader/public_tests/OkHttp3DownloaderPublicTest.cs
using System;
using Xunit;

using Picasso;

namespace PicassoTests.Public
{
    public class OkHttp3DownloaderPublicTest
    {
        [Fact]
        public void TestShutdownIsSafeRepeatedly()
        {
            var downloader = new OkHttp3Downloader();
            downloader.Shutdown();
            downloader.Shutdown();
        }

        [Fact]
        public void TestAnotherDummyConstructor()
        {
            var downloader = new OkHttp3Downloader("publicValue");
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/JakeWharton_picasso2-okhttp3-downloader/public_tests/okhttp3downloader_public_test.go
package public_tests

import (
	"testing"

	"github.com/jakewharton/picasso2-okhttp3-downloader/picasso"
)

func TestShutdownIsSafeRepeatedly(t *testing.T) {
	downloader := picasso.NewOkHttp3Downloader()
	downloader.Shutdown()
	downloader.Shutdown()
}

func TestAnotherDummyConstructor(t *testing.T) {
	_ = picasso.NewOkHttp3DownloaderWithDummy("publicValue")
}